rpm/archivematica-storage-service/package.spec

```diff
@@ -76,12 +76,17 @@ rm -rf %{buildroot}
 
 %post
 
-# Create archivematica user
-userID=`id -u archivematica`
-if [ "${userID}" != 333 ]; then
-  useradd --uid 333 --user-group --home /var/lib/archivematica/ archivematica
+# Create archivematica user and group
+getent group archivematica >/dev/null || groupadd -f -g 333 -r archivematica
```

Comment:
I've seen that this snippet comes from http://fedoraproject.org/wiki/Packaging%3aUsersAndGroups#Soft_static_allocation but I don't understand why we need `getent group archivematica >/dev/null || ` considering that `groupadd -f` is going to exit successfully regardless. Can you think of a reason? I've read their rationale but still it doesn't make sense in this case.

```
$ docker run -it centos:7 bash
% groupadd -g 12345 -r archivematica
% groupadd -f -g 333 -r archivematica
% getent group 333
% getent group 12345
archivematica:x:12345:
```